tests/helpers/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { browser } from '@wdio/globals';
+import type { EditorView, Workbench } from 'wdio-vscode-service';
+import type { ExtensionPageHandler } from '../types/helpers.ts';
+
+export async function waitUntilExtensionLoaded(): Promise<ExtensionPageHandler> {
+	const welcomePageTitle = 'Welcome to GitLens';
```

Comment:
could be fine to reuse from code 
https://github.com/gitkraken/vscode-gitlens/blob/c3a35b4362f153454541a4aea435b3d2f9f05bac/src/webviews/welcome/registration.ts#L13

Reply:
Tried this but I'm finding some problems with ES and JS configuration. Let's add this as an improvement, shall we?